cypress/support/commands/rancher-api-commands.ts

```diff
@@ -14,15 +14,18 @@ Cypress.Commands.add('login', (
   username = Cypress.env('username'),
   password = Cypress.env('password'),
   cacheSession = true,
+  skipNavigation = false,
 ) => {
   const login = () => {
     cy.intercept('POST', '/v3-public/localProviders/local*').as('loginReq');
 
-    LoginPagePo.goTo(); // Needs to happen before the page element is created/located
+    if (!skipNavigation) {
+      LoginPagePo.goTo(); // Needs to happen before the page element is created/located
+    }
```

Comment:
Maintained the expected default behavior. This was needed for the login landing page tests. Without it we would logout and when attempted to login we'd get redirected to a brand new login page. This meant that we would lose the record of what page the user was on before getting redirected to the login page.